lsp-mode.el

```diff
@@ -6015,20 +6015,24 @@ WORKSPACE is the active workspace."
 
 (defmacro lsp--read-json (str)
   "Read json string STR."
+  `(let ((json-object-type (if lsp-use-plists
```

Comment:
can we go without these changes? Using the global variables will result in a slow down. It is not big deal I guess but it will still be better to avoid that considering this is on the hot path.

Reply:
Done